pages/u.tsx

```diff
@@ -23,7 +23,7 @@ const Profile = ({ repo_list }: ProfileProps) => {
 	const session: Session | null = useSession().data;
 	useEffect(() => {
 		rudderEventMethods().then((response) => {
-			response?.track(`${getAuthUserId(session)}`, "Repo Profile Page", {"userId": `${getAuthUserId(session)}`, "name": getAuthUserName(session)}, `${null}`);
+			response?.track(`${getAuthUserId(session)}`, "Repo Profile Page", { "userId": `${getAuthUserId(session)}`, "name": getAuthUserName(session) }, `${null}`);
 		});
```

Comment:
This has been modified. But we can still push it to production as the modified changes are still in review.

Reply:
Also, you can revert all the changes regarding datatype change of `userId`  maybe? Since, meri PR already ye address karti hain and waha I have even added code to add `userId` manually using `uuidv4()` method at the time of signing up.